log-collector/src/test/scala/io/phdata/pulse/logcollector/SolrCloudStreamTest.scala

```diff
@@ -21,6 +21,8 @@ import io.phdata.pulse.solr.{ BaseSolrCloudTest, TestUtil }
 import org.apache.solr.client.solrj.SolrQuery
```

Comment:
Make sure to remove all unused imports

Reply:
I have added the testcases in the existing test suite. `SolrQuery` is already used in line-64 and 87.

Reply:
oh you need to delete the ignored tests